src/btree/bt_split.c

```diff
@@ -815,7 +815,8 @@ __wt_multi_to_ref(WT_SESSION_IMPL *session,
  */
 static int
 __split_parent(WT_SESSION_IMPL *session, WT_REF *ref,
-    WT_REF **ref_new, uint32_t new_entries, size_t parent_incr, int exclusive)
+    WT_REF **ref_new, uint32_t new_entries, size_t parent_incr, int exclusive,
```

Comment:
Normally if there are more than two flag parameters we'd turn them into a single flags. See:
https://github.com/wiredtiger/wiredtiger/blob/develop/src/btree/bt_walk.c#L17
For a function with a flags argument
https://github.com/wiredtiger/wiredtiger/blob/develop/src/evict/evict_lru.c#L1541
For an example of calling such a function.

You can define the flags you need in ```dist/flags.py```, and their values will be auto-generated.